packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -130,7 +132,7 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
                 );
               },
             ),
-            const SizedBox(width: denseSpacing),
+            SizedBox(width: isDense() ? denseModeDenseSpacing : denseSpacing),
```

Comment:
revert this padding back to dense spacing. We can add a line to IconLabelButton that will help with the width problem.